app/controllers/api/v2/base_controller.rb

```diff
@@ -64,6 +65,10 @@ def metadata_total
         @total ||= resource_scope.try(:count).to_i
       end
 
+      def can_create?
+        resource_class.include?(Authorizable) && User.current.can?("create_#{controller_permission}")
```

Comment:
Not all resources includes Authorizable module. In fact they do only if they support granular filtering. In this case, I htink it would be safer to use `authorized_for` helper and pass it the url, which already contains this logic. I'll be happy to explain permission system a bit more in details in person if that helps.

Reply:
Sure. No problem!
Thanks for your review.